### DISCUSSION TITLE
Adjust image saturation

### DISCUSSION THREAD

**Discussion by LuckyLuke-n** [dlemstra/Magick.NET#1272] [Help] [answered]
Hi all,
while searching through the methods in this library I found possibilities to adjust brightness and contrast but not for saturation.
Did I miss something here? Is there another way to adjust the saturation? Sorry I am quite new to the whole working on images thing.
Thank you very much in advance.

**Comment by dlemstra** [comment, marked as answer]
You can change the saturation with the `Modulate` method.

**Reply by LuckyLuke-n** [reply]
Thank you.
Unfortunately, I didn't check thoroughly enough about what I needed. I have to change the Lightness, not the Saturation.  Would you kindly give my another hint here?
Is there some sort of documentation where I could find out this without bothering people here?

**Reply by dlemstra** [reply]
Maybe you can take a look here: https://imagemagick.org/Usage/basics? Most options are also available as a method on the `MagickImage` class.
